.github/workflows/synchronizing.yaml

```diff
@@ -96,8 +96,8 @@ jobs:
 
       - name: apply changes to the files
         run: |
-          if echo "${{ github.event.head_commit.message }}" | grep -q "#"; then
-            branch_name=$(echo "${{ github.event.head_commit.message }}" | grep -o '[0-9]\+')
+          if echo "${{ github.event.head_commit.message }}" | grep -q '#\d*'; then
```

Comment:
For '\d' perl regex, should use ```grep -P``` or [0-9] character class. Tested on Alinux. Mac grep works different.